docker/official/Dockerfile

```diff
@@ -1,8 +1,11 @@
 # This image is based off the latest Jenkins LTS
-FROM jenkins:alpine
+FROM jenkinsci/jenkins:lts-alpine
```

Comment:
A more current and correct LTS too, `jenkinsci/jenkins` is always updated automatically after releases. Whereas the "official library" image (`jenkins`) requires a manually submitted and approved pull request :face_with_head_bandage: